packages/devtools_app/test/app_size/app_size_screen_test.dart

```diff
@@ -170,8 +170,10 @@ void main() {
       expect(breadcrumbs.length, 1);
       expect(breadcrumbs.first.text, equals('Root [6.0 MB]'));
       expect(find.byType(BreadcrumbNavigator), findsOneWidget);
-      expect(find.text('package:flutter'), findsOneWidget);
-      expect(find.text('dart:core'), findsOneWidget);
+      String text = 'package:flutter';
```

Comment:
Small nit: can we switch back to directly passing the raw string to the finder (instead of setting as a variable). Here and elsewhere.